backend/agents/persona_agent.py

```
@@ -256,7 +330,7 @@ def get_persona_knowledge(query: str) -> str:
                 else:
                     raise ValueError("PGVector not available - vectorstore is required")
             except Exception as e:
-                print(f"Error in get_persona_knowledge: {e}")
+                debug_log(f"Error in get_persona_knowledge: {e}")
                 raise e
```

Ruff (0.14.1)

Use `raise` without specifying exception name

Remove exception name

(TRY201)